.github/workflows/build.yml

```diff
@@ -19,6 +19,7 @@ on:
     tags:
       - '**'
   pull_request:
+    types: [opened, reopened, ready_for_review]
```

Comment:
Unless I'm mistaken this disables rebuilds when someone pushes to the PR? Was that your goal?

Reply:
yes, that was the intent, since we also have a `on:push:branches:'**'` it triggers two builds one for the PR and the other one for the branch ...
an alternative can be that we explicitly list the branches we want to build, `main` and all the version branches (`[0-9]*.[0-9]*` would such a pattern work?)

Reply:
But when we push to a branch that has a pending PRs, most of the time that branch is in a fork repo... so that other build on the branch in a fork is not authoritative, we must ignore it.

As for dependabot I think I had this covered.

Reply:
ohh ok! 
I was testing it here https://github.com/marko-bekhta/hibernate-search/pull/174  in the fork, so yeah ... 😃